Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -217,6 +197,10 @@ void JitArm64::GenerateAsm()
   MOVP2R(ARM64Reg::X8, &CoreTiming::GlobalAdvance);
   BLR(ARM64Reg::X8);
 
+  // GlobalAdvance checks exceptions so
+  // set the membase here
```

Comment:
There's a second reason why this is important too: In case we just entered the JIT.